### ISSUE TITLE
Fuses and breakers to protect power lines.

### ISSUE THREAD

**Issue by Diamantino-Op** [GTNewHorizons/GT-New-Horizons-Modpack#17161]
### Your GTNH Discord Username

diamantinoop

### Your Pack Version

Nigtly 614

### Your Proposal

Add fuses and breakers to prevent accidental explosions from connecting a higher voltage output to a power line and overcurrent protection.

### Your Goal

Players won't be punished from making the error of connecting a high voltage source to lower voltage wires / machines, and won't accidentally burn wires if requesting too much current.

### Your Vision

Players won't lose half of their base when making this error.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this feature please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I believe there is nothing similar in the pack already, or the existing solution isn't good enough.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members thinking it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by ExileWeirdo** [comment]
Rebuilding half your base is fun tho! Idk i feel like explosions etc are a part of gtnh and has been since the start. Learning from mistakes and being super careful is why gtnh is what it is today. I think it wouldn't be a good idea to implement this because i like that gtnh is realistic with how things work to a certain point. If you send too much power into smt it should explode if u cant handle this.

Either way most players already have a fix to this and thats disabling explosions in Server Utilities or other mods you have so ig this would be cool but i dont think its really nessecary (i cant spell this word)

**Comment by FourIsTheNumber** [comment]
> Rebuilding half your base is fun tho! Idk i feel like explosions etc are a part of gtnh and has been since the start. Learning from mistakes and being super careful is why gtnh is what it is today. I think it wouldn't be a good idea to implement this because i like that gtnh is realistic with how things work to a certain point. If you send too much power into smt it should explode if u cant handle this.
> 
> Either way most players already have a fix to this and thats disabling explosions in Server Utilities or other mods you have so ig this would be cool but i dont think its really nessecary (i cant spell this word)

It's not realistic at all, fwiw. What they're proposing is being able to optionally build safety measures, which imo is potentially interesting - if player can optionally choose to invest in and use safety tools.

**Comment by Diamantino-Op** [comment]
Yep, that's exactly what i was proposing, if a player likes the risk he can just not build safety measures, but if you don't want to lose machines because of an error, you can use those to protect them.

**Comment by ExileWeirdo** [comment]
thats true, ig if people dont wanna be "cheaty" by using server utilities its a cool concept. depends on how cool the fuses could be n stuff. How would it integrate with all different tectech stuff and so? would it just delete the explosion or? Would be cool if it was tiered, like t1 fuse would minimize 50% of the explosion, t2 70% etc

**Comment by OmdaCZ** [comment]
Don't diodes have this function already? They should blow up before any machines due to being before them and they limit amperage to not burn down cables.

**Comment by Diamantino-Op** [comment]
for fuses i was thinking of cables, like if the cable is transferring more than the max allowed, the fuse would burn before all the other cables.

**Comment by Diamantino-Op** [comment]
> Don't diodes have this function already? They should blow up before any machines due to being before them and they limit amperage to not burn down cables.

well, that's true, but what if it is in the main powerline or a line without diodes?

**Comment by Diamantino-Op** [comment]
like a line with a flow of 64+ amps of power
